services/node/node-base.js

```diff
@@ -15,6 +15,14 @@ module.exports = class NodeVersionBase extends NPMBase {
 
   static get examples() {
     const type = this.type
+    const documentation = `
+  <p>
+    This badge is designed for projects that release packages to <a href="https://www.npmjs.com/">npm</a><br/>
+    The badge can be added to the project readme to inform users of the node versions that the package is supported by<br/>
+    The information is retrieved from the <code>engines.node</code> section in package.json<br/>
+    ${this.documentation}
+  </p>
```

Comment:
Thanks for following up with these! One thing to keep in mind with the inline badge help text is that there's a general assumption that once Shields users get to this point they are generally familiar with the corresponding service and the basic premise of Shields and utilizing badges.

The inline help text is primarily geared towards things like providing guidance on how users can locate or retrieve the parameter values (for example how to find your project id), explaining the difference between badge A and similarly-looking badge B, etc.

In this context, the most important thing to convey IMO is the distinction between the LTS badge and the current badge, and if possible with links out to relevant info on Node's release cycle. I know the last sentence in the text speaks to that, but I don't think that point is emphasized as much as it could and should be.

If you feel strongly that the additional information is important, I'd still suggest tweaking the text/structure of the text to get a better presentation of the material.

In the not-so-distant past we changed the help text alignment from centered to left which works best in most circumstances, but not so much in cases like this with multiple blocks of text.

Lastly, remember that these badges can work with any custom registry as well, not just the main npmjs registry. As such the current text of that first sentence could be a point of confusion

![image](https://user-images.githubusercontent.com/13042488/78503137-582e9100-772a-11ea-9ff3-e963218b1593.png)